pr/okta-oidc-doc-changes (#4593)
Kong/docs.konghq.com: src/gateway/kong-plugins/authentication/oidc/okta.md

Configure the following parameters:

> __In the common tab__

Reviewer: ```suggestion
From the **Common Tab**:
```

* **`issuer`**: The issuer `url` from which OpenID Connect configuration can be discovered. Using Okta, specify the domain and server in the path:
    * `https://YOUR_OKTA_DOMAIN/oauth2/YOUR_AUTH_SERVER/.well-known/openid-configuration`
* **`auth_method`**: A list of authentication methods to use with the plugin, such as passwords, introspection tokens, etc. The majority of cases use `authorization_code`, and {{site.base_gateway}} will accept all methods if no methods are specified.
* **`client_id`**: The `client_id` of the OpenID Connect client registered in OpenID Connect Provider. Okta provides one to identify itself.  
* **`client_secret`**: The `client_secret` of the OpenID Connect client registered in OpenID Connect Provider. These credentials should never be publicly exposed.

> __In the authorization tab__

Reviewer: ```suggestion
From the **Authorization Tab**
```

> * **`Config.Scopes Required`**:  The scope of what OpenID Connect checks. `openid` by default; set to `email` and `profile` for this example. 

Reviewer: ```suggestion
* **`Config.Scopes Required`**:  The scope of what OpenID Connect checks, this value is set to `openid` by default. The example is set to `email` and `profile`.
```

* **`Config.Scopes Claim`**: This should be set to `scp`.

> __In the advanced tab__

Reviewer: ```suggestion
From the **Advanced Tab**
```

> * **`Config.Redirect Uri`**: The `redirect_uri` of the client defined with `client_id` (also used as a redirection URI for the authorization code flow).

Reviewer: ```suggestion
* **`Config.Redirect URI`**: The `redirect_uri` of the client defined with `client_id` (also used as a redirection URI for the authorization code flow).
```
